client/src/App.tsx

```diff
@@ -1,38 +1,195 @@
 import "./App.scss";
+import { useState, useRef } from "react";
 import PlayerScore from "./components/PlayerScore/PlayerScore";
-import { DEFAULT_ROUNDS, DEFAULT_PLAYERS } from "./constants/constants";
-import { Header } from "./components/header/Header";
-import { initialiseGame } from "./service/game";
-import { useState } from "react";
+import {
+  DEFAULT_ROUNDS,
+  DEFAULT_PLAYERS,
+  DEFAULT_TIMEOUT,
+} from "./constants/constants";
+import { Header } from "./components/Header/Header";
+import { initialiseGame } from "./service/game/game";
 import { Game } from "./types/game/game.types";
+import { Player } from "./types/player/player.types";
+import { Stat } from "./types/card/card.types";
+import {
+  chooseRandomStat,
+  calculateRoundWinner,
+  updateRoundWinners,
+  updatePlayerScores,
+  updatePlayerCards,
+  updatePlayerIsCardShown,
+  updatePlayerIsCardShownAll,
+} from "./service/round/round";
+
+//Game starts with two players each with five cards
+//Clicking "Play" triggers startRound to start the first round (and the game)
+//showPlayerCard reveals the top card in the current player's hand
+
+//If the player !isHuman there is a setTimeout to mimic a human choosing a stat
+//chooseRandomStat selects a random stat from the current player's card
+//playRound then runs with the chosen stat
+
+//If the player isHuman there should be a prompt to pick a stat from the card i.e. "Please chose a stat player 2"
+//The player clicks on a stat which triggers playRound to run with their chosen stat
+
+//showAllCards reveals all of the player's top cards
+//A setTimeout is used to allow time for all the cards to be shown
+//calculateRoundWinner then compares the value of chosen stat with the value of that stat on the top card of every other player
+//The player with the card with the highest value for that characteristic wins the round
+//If there is a draw between multiple highest values then the current player wins the round
+
+//endRound then runs to end the round and prepare for the next round
+//hideAllCards turns all the cards face down
+//A setTimeout is used to allow time for all of the cards to be hidden
+//The top card, i.e. the card used this round, for each player is set to the bottom of their hand
+
+//If the currentRoundRef === game.totalRounds then the game ends and a winner is declared
+//If the curerntRoundRef !== game.totalRounds then startRound() runs and to start the round for the next player
+
```

Comment:
I assume this will go once we've got the logic all in fine?

Reply:
Yep, this started as pseudo code so I could work out what I was doing, but I left it in to help with reviewing. Will be deleted once we're happy with the round/game logic